kroxylicious-filter-archetype/src/main/resources/archetype-resources/src/test/java/SampleFilterIT.java

```diff
@@ -8,207 +8,95 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 
 import io.kroxylicious.proxy.config.ConfigurationBuilder;
 import io.kroxylicious.proxy.config.NamedFilterDefinitionBuilder;
-import io.kroxylicious.test.tester.KroxyliciousTester;
+import io.kroxylicious.test.tester.KroxyliciousConfigUtils;
+import io.kroxylicious.test.tester.KroxyliciousTesters;
 import io.kroxylicious.testing.kafka.api.KafkaCluster;
 import io.kroxylicious.testing.kafka.common.BrokerCluster;
 import io.kroxylicious.testing.kafka.junit5ext.KafkaClusterExtension;
 import io.kroxylicious.testing.kafka.junit5ext.Topic;
 
-import static io.kroxylicious.test.tester.KroxyliciousConfigUtils.proxy;
-import static io.kroxylicious.test.tester.KroxyliciousTesters.kroxyliciousTester;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.fail;
+import static org.assertj.core.api.Assertions.assertThat;
 
 @ExtendWith(KafkaClusterExtension.class)
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
 class SampleFilterIT {
 
-    // Configure filters here
-    private static final String FIND_CONFIG_FIELD = "findValue";
-    private static final String REPLACE_CONFIG_FIELD = "replacementValue";
-    private static final TestFilter SAMPLE_PRODUCE_REQUEST_FILTER = new TestFilter(SampleProduceRequest.class.getName(),
-            Map.of(FIND_CONFIG_FIELD, "foo", REPLACE_CONFIG_FIELD, "bar"));
-    private static final TestFilter SAMPLE_FETCH_RESPONSE_FILTER = new TestFilter(SampleFetchResponse.class.getName(),
-            Map.of(FIND_CONFIG_FIELD, "bar", REPLACE_CONFIG_FIELD, "baz"));
+    private static final Map<String, Object> CONSUMER_CONFIGURATION =
+            Map.of(ConsumerConfig.GROUP_ID_CONFIG, "group-id-0",
+                    ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
 
-    // Configure test input/expected values here
-    private static final String NO_TRANSFORM_VALUE = "sample";
-    private static final String PRE_TRANSFORM_VALUE = "foo bar baz";
-    private static final String FETCH_TRANSFORM_VALUE = "foo baz baz";
-    private static final String PRODUCE_TRANSFORM_VALUE = "bar bar baz";
+    private static final int TIMEOUT = 10;
 
-    // Configure Cluster/Producer/Consumer values here
-    private static final Integer TIMEOUT_SECONDS = 10;
+    private static final Map<String, Object> FILTER_CONFIGURATION =
+            Map.of("findValue", "foo", "replacementValue", "bar");
 
     @BrokerCluster
-    KafkaCluster cluster;
+    private KafkaCluster kafkaCluster;
 
-    Topic topic; // Injected by KafkaClusterExtension
+    @SuppressWarnings("unused")
+    private Topic topic;
 
-    FilterIntegrationTest test;
+    private ConfigurationBuilder proxyConfiguration;
 
-    @AfterEach
-    public void afterEach() {
-        test.close();
+    @BeforeEach
+    public void setUp() {
+        this.proxyConfiguration = KroxyliciousConfigUtils.proxy(kafkaCluster);
     }
```

Comment:
I think it would simplify the test if we convert the `kafkaCluster`, `topic` from fields into method parameters and build the `proxyConfiguration` in each test rather than use a field/setUp.